.swiftlint.yml

```diff
@@ -0,0 +1,8 @@
+disabled_rules:
+  - trailing_comma
```

Comment:
This rule enforces trailing commas in dictionaries and lists, right? We actually want that.

Reply:
Unfortunately, no. It warns when there is a trailing comma.

[trailing_comma](https://realm.github.io/SwiftLint/trailing_comma.html)

Reply:
@cgrindel @brentleyjones You can configure `trialing_comma` to instead enforce that there is a trailing comma. You need to add a config statement for the rule though.

```yaml
trailing_comma:
  mandatory_comma: true
```

Reply:
Cool. I did not know that. I will give it a try and put up a new PR. Thanks!